source/content/guides/decoupled/overview/05-manage-settings.md

```diff
@@ -31,13 +31,36 @@ You can change the Site Name, connect or disconnect your Git repository, and del
 
 ### Node.js Version
 
-Pantheon respects the setting in `.nvmrc` when selecting the Node.js version for runtime. Currently supported versions are:
+As of the release of our [V1 build pipeline](/guides/decoupled/overview/v1-pipeline), Front-End Sites runs can run v16, v18, or v20 of Node.js as specified in your `package.json` file.
+
+Non-LTS versions like v17 are not available.
```

Comment:
> Non-LTS versions like v17 are not available.

I suggest removing this sentence or revising it for accuracy based on https://nodejs.org/en/about/previous-releases  (which suggests v16 is also Non-LTS ?) 

If revising, I suggest adding context for "Non-LTS" maybe in a [tooltip](https://docs.pantheon.io/style-guide#tooltips)